packages/header-component/src/components/dc-header/dc-dropdown.css

```diff
@@ -3,6 +3,10 @@
   display: none;
 }
 
+:host .dropdown-label {
+  font-family: var(--main-font);
```

Comment:
Why we don't apply this to the root of the component and avoid to "cherry-pick" which node needs the font?
Is there any overwrite for this? For instance I would expect something like:

```
:host header {
    font-family: var(--main-font);
}
```

_Assume header is a class that behaves as the container of the whole component_